components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8192,8 +8243,9 @@ def getZoomLevelScaling(self):
             rv[i] = float(level)/sizeXList[0]
         return rv
 
+    @assert_re()
     def setActiveChannels(self, channels, windows=None, colors=None,
-                          reverseMaps=None):
+                          reverseMaps=None, noRE=False):
```

Comment:
@joshmoore conflict were on `noRE` as `reverseMaps` were added in develop. This is something to review